BlogTemplate/Models/ExcerptGenerator.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BlogTemplate.Models
+{
+    public class ExcerptGenerator
+    {
+        private BlogDataStore _dataStore;
+
+        public ExcerptGenerator(BlogDataStore dataStore)
+        {
+            _dataStore = dataStore;
+        }
+
+        public string CreateExcerpt(string body)
```

Comment:
You could make this method static since it only depends on its inputs, so that you don't need to create a new instance.